screens/MessagesUsers/index.js

```diff
@@ -1,17 +1,25 @@
 import {useEffect, useState} from 'react';
 import PropTypes from 'prop-types';
-import {FlatList, Text, TouchableOpacity, View} from 'react-native';
+import {Pressable, Text, TouchableOpacity, View} from 'react-native';
 import SkeletonMessagesList from '../../components/Skeleton/SkeletonMessagesList';
 import Warning from '../../components/Warning/Warning';
 import User from './User';
 import Search from '../../components/Search/Search';
+import DeleteIcon from '../../assets/icons/Delete.svg';
 import {getDataUsersFromFile} from '../../utils/ApiUtils';
+import SwipeableFlatList from 'react-native-swipeable-list';
+import {theme} from '../../assets/theme/theme';
+import PermissionModal from '../../components/Permission/Modal';
+import UsersMessagesModal from '../../components/Permission/children/remove';
 import styles from './style';
 
 const MessagesUsers = ({navigation}) => {
   const [loading, setLoading] = useState(true);
   const [data, setData] = useState([]);
   const [state, setState] = useState(data);
```

Comment:
state is not good name

Reply:
also data state can't be default value for another state

Reply:
I will fix it.